src/core/annotation.js

```diff
@@ -791,6 +794,58 @@ class MarkupAnnotation extends Annotation {
   setCreationDate(creationDate) {
     this.creationDate = isString(creationDate) ? creationDate : null;
   }
+
+  _setDefaultAppearance(xref, drawColor, fillColor, blendMode, pointsCallback) {
```

Comment:
`drawColor` -> `strokeColor`, since the latter is consistent with both the specification and the PDF.js implementation.

Also, given the number of parameters the readability of the call-sites are suffering quite a bit in my opinion, hence changing this method to take a parameter object instead would really help:
```suggestion
  _setDefaultAppearance({
    xref,
    strokeColor,
    fillColor,
    blendMode,
    pointsCallback,
  }) {
```